tests/robot_tests.cpp

```diff
@@ -0,0 +1,53 @@
+#include <pick_ik/robot.hpp>
+
+#include <catch2/catch_all.hpp>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
+
+#include <moveit/utils/robot_model_test_utils.h>
+
+using moveit::core::RobotModelBuilder;
+
+namespace pick_ik {
```

Comment:
Why open a namespace in the tests?

Reply:
so I don't type `pick_ik::` all through my tests, makes it easier to read.

Reply:
I'd still prefer we always spell out the namespace of the library under test. It makes it obvious when we're testing something from the library and it gives us a better idea for what it's like to use our namespace in the real world. If we're so eager to not type out that namespace, perhaps a shorter namespace is in order.

Reply:
pushed a commit that fixes this